payment_payfip/controllers/main.py

```diff
@@ -2,63 +2,63 @@
 import pprint
 import werkzeug
 
-from odoo import _, http
+from odoo import http, exceptions
```

Comment:
```suggestion
from odoo import exceptions, http
```
Nous mettons dans l'ordre alphabétique par convention Horanet, cela peut s'appliquer pour l'ensemble des fichier python de ce projet